mesh_aws_client/mesh_send_message_chunk_application.py

```diff
@@ -110,7 +110,6 @@ def start(self):
 
         is_finished = self.input.get("complete", False)
         if is_finished:
-            # TODO log error
             self.response.update({"statusCode": HTTPStatus.INTERNAL_SERVER_ERROR.value})
             raise SystemError("Already completed upload to MESH")
```

Comment:
Critical/ Error log reference missing

Reply:
Raising 'SystemError' will create a critical log from the logging infrastructure.